src/Entity/CitizenAction.php

```diff
@@ -52,9 +44,9 @@ public function __construct(
         $this->category = $category;
         $this->description = $description;
         $this->postAddress = $address;
-        $this->capacity = $capacity;
         $this->participantsCount = $participantsCount;
-        $this->beginAt = $beginAt;
+        // We need a \DateTime object for now to work with Gedmo sluggable
+        $this->beginAt = $beginAt instanceof \DateTimeImmutable ? new \DateTime($beginAt->format(\DATE_ATOM)) : $beginAt;
```

Comment:
Just Use https://secure.php.net/manual/en/datetimeimmutable.createfrommutable.php instead of `new \DateTime`

Reply:
For the record, I misundertood it's ok :-)